src/main/java/io/jenkins/plugins/report/jtreg/main/comparator/Arguments.java

```diff
@@ -7,85 +7,72 @@ public static Options parse(String[] arguments) {
 
         if (arguments.length >= 1) {
             for (int i = 0; i < arguments.length; i++) {
-                switch (arguments[i].toLowerCase()) {
-                    case "-h":
-                    case "--help":
+                String currentArgument = arguments[i];
```

Comment:
just a hint. Using case isnensitive arguemtns usually do not bring any benefits.  I think all arguments[i].toLowercase may be dropped.